Source/LinqToDB/Expressions/ExpressionEqualityComparer.cs

```diff
@@ -34,327 +34,310 @@ public sealed class ExpressionEqualityComparer : IEqualityComparer<Expression>
 		public int GetHashCode(Expression? obj)
```

Comment:
maybe this and update all calls to GetHashCode  in GetHashCode by calls to UpdateHashCode?
```suggestion
		public int GetHashCode(Expression? obj)
		{
			var hashCode = new HashCode();
			UpdateHashCode(ref hasCode, obj);
			return hasCode;
		}
		
		void UpdateHashCode(ref HashCode hashCode, Expression? expression)
```